backend/serverless.yml

```diff
@@ -215,6 +215,32 @@ functions:
           path: /api/v1/auth/lti
           method: post
           cors: true
+  # Step 2: Authentication Request
+  login:
+    handler: src/handlers/auth/lti.login
+    timeout: 10
+    events:
+      - http:
+          path: /api/v1/auth/login
```

Comment:
Fix the paths to follow the following convention

`api/v1/auth/{PLATFORM}/{ACTION}`